platform/overlays/gke/istio.yaml

```diff
@@ -6731,12 +6731,18 @@ metadata:
 spec:
   loadBalancerIP: 34.95.5.243
   ports:
+  - name: status-port
+    port: 15021
+    targetPort: 15021
```

Comment:
These three lines and three for port 15443 should be removed. This exposes health check data and shows up in port scans which is a little gross. I don't have an automated way of removing this yet, but I'll figure out something soon.